models/dodo/arbitrum/dodo_arbitrum_schema.yml

```diff
@@ -168,8 +162,7 @@ models:
             - version
             - tx_hash
             - evt_index
-            - trace_address
-      - check_dex_aggregator_seed_legacy:
```

Comment:
these should be reverted, as the spell is out of scope for this PR (it's `dex_aggregator.trades`)